docs/CONTRIBUTING.md

```diff
@@ -102,3 +111,8 @@ To regenerate the golden files, run the tests with the `generateSqlFiles` Gradle
 ```
 ./gradlew cleanTest test --tests bio.terra.tanagra.* --info -PgenerateSqlFiles=true
 ```
+
```

Comment:
It looks like there's a `.CONTRIBUTING.md.swp` binary file that also got pushed -- do you need that one?